pom.xml

```diff
@@ -104,6 +104,11 @@
             <artifactId>lombok</artifactId>
             <version>1.16.16</version>
         </dependency>
+        <dependency>
+            <groupId>com.fasterxml.jackson.jr</groupId>
+            <artifactId>jackson-jr-objects</artifactId>
+            <version>2.8.8</version>
+        </dependency>
```

Comment:
https://github.com/FasterXML/jackson-jr
* the main value add for this is a smaller footprint.  and it is at best as fast as jackson-databind
We should continue to use jackson-databind.

A shared instance of ObjectMapper is fine, since we dont change serialization config. Or we can use the immutable ObjectWriter.   see http://stackoverflow.com/questions/3907929/should-i-declare-jacksons-objectmapper-as-a-static-field